source/features/more-dropdown.tsx

```diff
@@ -33,17 +33,19 @@ export function onlyShowInDropdown(id: string): void {
 	select('.js-responsive-underlinenav-overflow ul')!.append(menuItem);
 }
 
-async function init(): Promise<void> {
+export async function unhideOverflowDropdown(): Promise<void> {
```

Comment:
@cheap-glitch can you think of a better function name?

Reply:
Maybe `alwaysShowDropdown` or `showOverflowDropdown`? (but I think it's fine as is :+1:)